subprojects/gradle/instrumentation-test-impact-analysis/src/main/kotlin/com/avito/instrumentation/impact/TestBytecodeAnalyzeAction.kt

```
@@ -105,10 +105,13 @@ abstract class TestBytecodeAnalyzeAction : WorkAction<TestBytecodeAnalyzeAction.
 
     override fun execute() {
         val contextLoader = ContextLoader()
-        val foldersWithClassesToLoad = bytecodeResolver.resolveBytecodeWithoutDependencyToAnotherConfigurations(
-            reportType = ReportType.ANDROID_TESTS
+        val androidTestsBytecodeFolders = bytecodeResolver.resolveBytecodeWithoutDependencyToAnotherConfigurations(
+            configurationType = ConfigurationType.ANDROID_TESTS
         )
-        val context: Context = contextLoader.load(foldersWithClassesToLoad)
+        require(androidTestsBytecodeFolders.isNotEmpty()) {
```

fail fast with a clear message